src/lib/injections.js

```diff
@@ -73,10 +75,31 @@ class Injections {
   assignContentScriptDefaults(contentScripts) {
     let finalConfig = Object.assign({}, contentScripts);
 
+    // Don't persist the content scripts across browser restarts
+    // (at least not yet, we would need to apply some more changes
+    // to adjust webcompat for accounting for the scripts to be
+    // already registered).
+    finalConfig.persistAcrossSessions = false;
+
     if (!finalConfig.runAt) {
       finalConfig.runAt = "document_start";
     }
 
+    if (!finalConfig.id) {
+      finalConfig.id = `generated-id-${++lastScriptId}`;
```

Comment:
The `id` property should always be there, [as there is a test that would fail if it's unset](https://github.com/mozilla-extensions/webcompat-addon/blob/16f6e7b55a6d4e8ee7c8383ad6aedb87d84d426f/spec/available_injections.spec.js#L39-L41).

Also, [there's a test that fails if the id isn't unique](https://github.com/mozilla-extensions/webcompat-addon/blob/16f6e7b55a6d4e8ee7c8383ad6aedb87d84d426f/spec/available_injections.spec.js#L19-L34), and while that only applies within each "type" (i.e. there could be a UA override with the same ID as an injection), any injection with a duplicate iD would already fail CI.

Reply:
yep, I was meant to actually double-check that I could use the `id` property of the injection instead of a generated id, but I didn't yet in the previous version of the patch (I should have captured it more visibly in a TODO inline comment to be fair).

Covered as part of the last update on this PR.